core/src/main/resources/hudson/model/Computer/markOffline.jelly

```diff
@@ -30,6 +30,7 @@ THE SOFTWARE.
 
   <l:layout title="${%title(it.displayName)}">
     <st:include page="sidepanel.jelly" />
+    <l:breadcrumb title="${%title(it.displayName)}" />
```

Comment:
from what I can tell this would be `Taking EC2 (aws-us-east-2) - Windows 2019 (i-0f8ce0164c5fdd1f1) Offline` which is far too long
```suggestion
    <l:breadcrumb title="${%Mark offline}" />
```